src/composables/events/DialogEvent.js

```diff
@@ -0,0 +1,12 @@
+import { Subject } from 'rxjs';
+
+/**
+ * Represent an Event object to emit and to receive to close or open a dialog.
```

Comment:
`Represent an Event object to emit and to receive to close or open a dialog.`
-> `Represent an *rxjs* Event object to emit and to receive *events* to close or open a dialog.`

Reply:
done